docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`5572` Trade events in Uniswap V2 swaps now occur in a consecutive order.
```

Comment:
This is not a good description as it tells nothing to the user and also implies all events were wrong.

```suggestion
* :bug:`5572` Fix a detection error that caused some uniswap v2 trade edge cases not to be accounted for properly in pnl report.
```